addon/src/lib/frameScripts.js

```diff
@@ -1,63 +0,0 @@
-/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
removal of frame scripts, see https://github.com/mozilla/testpilot/pull/3456/files#r178343655